### ISSUE TITLE
Merge with intranet?

### ISSUE THREAD

**Issue by ethanresnick** [TechAtNYU/blueprint#11]
Why was blueprint created originally, rather than just adding new features to the intranet? I'm genuinely asking because I don't know. Was it because it was easier to code a separate project than to change the intranet's code? Or was there a design/organizational reason for blueprint to be separate?

From my POV, unless there are strong design reasons to do otherwise, it would make sense to try to keep everything in the intranet, for a few reasons:

1. Whenever we show data, there's a possibility that some data will be missing or outdated, and I think the user is much more likely to fix that missing/outdated data if they're already in a place where they know data editing is possible, like the intranet. E.g., imagine marketing is looking at a list of related clubs (to cross-promote with), and they think of one that's missing...I think they're more likely to add that club if they're in the intranet than in blueprint.

2. Long term, I think it's better if we only have one site that board members have to have to learn about. If there are two sites, people not only have to learn about the existence of both, but they have to remember which features are on which site, and most teams will have to switch between them.

3. Between having two separate, less-well-refined projects and one bigger, more-ingrained-in-the-organization application (like the intranet could become), I think having one bigger product increases the odds that that product will be cared for and evolved by future tech@nyuers, rather than each new group of board members trying to start from scratch. This is the issue I brought up [on discuss](https://discuss.techatnyu.org/t/lets-rewrite-less/150).

Thoughts?

**Comment by AbhiAgarwal** [comment]
I agree with all the three points above! This was actually on our to-do list for the semester.

I always wanted it to be a part of the Intranet, and to have one system that governs all. My blockage last semester was that I did not know how to architect it with the Intranet too well, and @maxdumas was busier than usual last semester. Going forward I would also like to merge the two projects into one. 

The only reason was because it was easier to make it a separate project than to change the Intranet's code. I attempted to play around with the Intranet's code, but realized that it was going to become a blocker for me so I shifted to just creating it as a new project (to view it as a test to see if it was useful information). I think @maxdumas is probably able to help me merge these two projects properly during this semester, though.

The design reasons you point out are definitely correct. I think having one system, in this case, is much better than having two separate sites. Another reason was I didn't know how to adapt the Intranet without having to re-think the design and the design elements on the Intranet. I would love to work on this, but it would have been another road-block and something that we never really sorted out even last year. 

That said! We should start doing this soon :+1:
